i18n-util/readme.md

```diff
@@ -1,7 +1,24 @@
 # i18n-util
 
   
-i18n-util package hosts utils to do locale computation and some custom redirection on help.okta.com. An alias url from the monolith in the format `help.okta.com/okta_help.htm#csh-eu-quick-access-apps?locale=ja` will get transformed into `help.okta.com/ja/prod/okta_help.htm#cshid=csh-eu-quick-access-apps`. `locale` is an optional parameter. If locale is not sent, we'll compute it based on a [priority](#locale)
+i18n-util package hosts utils to do locale computation and some custom redirection on help.okta.com. 
+The new help link base url format is now available for use, http://help.okta.com/okta_help.htm?id=${doc_alias}.
+
+Admin links will get redirected to en version of the docs since we don’t have translations yet.
+Example:
+
+http://help.okta.com/okta_help.htm?id=csh-config-asso
+
+End-user links will get redirected to the localized version of the docs since we have translations.
+Example:
+
+https://help.okta.com/okta_help.htm?id=csh_ov_overview?type=end-user
```

Comment:
nit: can we update this to &
`https://help.okta.com/okta_help.htm?id=csh_ov_overview**?**type=end-user`

Reply:
yes, good catch